bycycle/features/features.py

```diff
@@ -150,7 +146,8 @@ def compute_features(sig, fs, f_range, center_extrema='peak', burst_method='cycl
         raise ValueError('Invalid argument for "burst_method".'
                          'Either "cycles" or "amp" must be specified."')
 
-    if return_samples:
-        return df_features, df_samples
+    if return_samples is False:
```

Comment:
could / should this be a dataframe util? Just like we have 'split_samples_df', it could be `drop_samples_df` or something. I think it would be useful to have the utility if there may be some cases to do this operation outside this function.